app/views/admin/forms/index.html.erb

```diff
@@ -35,13 +35,13 @@
         Questions
       </th>
       <th data-sortable scope="col">
-        Activations
```

Comment:
activations isn't that useful

https://github.com/GSA/touchpoints/wiki/Analytics-Events gives more insight